src/mtool/ui/src/components/Header/index.js

```diff
@@ -480,6 +485,28 @@ class Header extends Component {
                 {this.props.OS_Running_Status}
               </Typography>
             ) : null}
+            <Tooltip
+              title={(
+                <>
+                  POS Last Active Time:
```

Comment:
Is it okay to use the word 'POS'? 

How about using 'PoseidonOS' which is the official name?

Reply:
yes, I also think the same. we are using PoseidonOS everywhere. let me change this. thanks